src/app/models.py

```diff
@@ -11,18 +11,30 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-
 from __future__ import unicode_literals
-
 from django.db import models
 from datetime import datetime
 from django import forms
 from django.contrib.auth.models import User
 
-
 class UserID(models.Model):
-    user = models.OneToOneField(User)
-    organisation = models.CharField("Organisation",max_length=64, null=False, blank=False)
-    lastlogin = models.DateField("Last Login",default=datetime.now,blank=True)
-    def __str__(self):
-        return self.user.username
+	user = models.OneToOneField(User)
+	organisation = models.CharField("Organisation",max_length=64, null=False, blank=False)
+	lastlogin = models.DateField("Last Login",default=datetime.now,blank=True)
+	def __str__(self):
+		return self.user.username
+
+class LicenseRequest(models.Model):
+	fullname = models.CharField(max_length=70)
+	shortIdentifier = models.CharField(max_length=25)
+	submissionDatetime = models.DateTimeField(auto_now_add=True)
+	userEmail = models.EmailField(max_length=35)
+
```

Comment:
Why aren't you adding all the fields in the models like you did in forms? Also, if you added all the field in the models, you can simply use `class Meta:` in forms. This will be a much more efficient way. Also, if there are any errors to the form submitted, you can easily check for them in forms.py . Let me know if you need help in this.